File: 01/src/main/java/Humanoid.java
public class Humanoid {
    private String humanoidType;

    public void walk() {
        if (humanoidType.equals("Lady with high heels")) {
            System.out.println("TOC! TOC! TOC!");
        } else {
            if (humanoidType.equals("Ninja")) {
                System.out.println("...");
            } else {
                if (humanoidType.equals("Pirate")) {
                    System.out.println("Tap, toc, tap, toc.");
                } else {
                    if(humanoidType.equals("Robot")) {
                        System.out.println("Bzz, bang, bzzz, bang");
                    } else {
                        if(humanoidType.equals("Regular human")) {
                            System.out.println("Tap, tap, tap, tap");
                        }
                    }
                }
            }
        }
    }

    public String getHumanoidType() {
        return humanoidType;
    }

    public void setHumanoidType(String humanoidType) {
        this.humanoidType = humanoidType;
    }
}

File: 01/dart/lib/humanoid.dart
class Humanoid {
  String humanoidType;

  void walk() {
    if (humanoidType == "Lady with high heels") {
      print("TOC! TOC! TOC!");
    } else {
      if (humanoidType == "Ninja") {
        print("...");
      } else {
        if (humanoidType == "Pirate") {
          print("Tap, toc, tap, toc.");
        } else {
          if (humanoidType == "Robot") {
            print("Bzz, bang, bzzz, bang");
          } else {
            if (humanoidType == "Regular human") {
              print("Tap, tap, tap, tap");
            }
          }
        }
      }
    }
  }
}
